flinkprocessor/src/main/java/io/pravega/example/videoprocessor/ImageResizer.java

```diff
@@ -43,13 +51,17 @@ public ImageResizer(int outputWidth, int outputHeight) {
 
     public void resize(InputStream inStream, OutputStream outStream) {
         try {
+            log.info("resize: BEGIN");
+            final long t0 = System.currentTimeMillis();
             BufferedImage inImage = ImageIO.read(inStream);
             Image scaledImage = inImage.getScaledInstance(outputWidth, outputHeight, Image.SCALE_SMOOTH);
             BufferedImage outImage = new BufferedImage(outputWidth, outputHeight, inImage.getType());
             Graphics2D g2d = outImage.createGraphics();
             g2d.drawImage(scaledImage, 0, 0, null);
             g2d.dispose();
             ImageIO.write(outImage, "png", outStream);
```

Comment:
shouldnt we be using jpeg?

Reply:
The output is a PNG because this can be encoded faster than JPEG and the output is expected to go to a subsequent task before it is stored. I will add a comment to the code to clarify this.